### ISSUE TITLE
Install fails without cython

### ISSUE THREAD

**Issue by richardjgowers** [MDAnalysis/mdanalysis#2337]
the `cython_linetrace` variable in setup.py is causing an error when Cython isn't installed.

More generally, the CI should be testing that the install without cython always works.... I think this is a little trickier than it seems because we only regenerate the .c files at release, rather than continually keep regenerating these at all states of the develop branch

**Comment by yurivict** [comment]
Same error in the FreeBSD port.
```
No openmp compatible compiler found default to serial build.
Will not attempt to use Cython.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 547, in <module>
    exts, cythonfiles = extensions(config)
  File "setup.py", line 324, in extensions
    if cython_linetrace:
NameError: global name 'cython_linetrace' is not defined
*** Error code 1
```

**Comment by richardjgowers** [comment]
fixed in 0.20.1
